src/Controller/Api/ContextController.php

```diff
@@ -255,7 +255,7 @@ protected function handleWriteContext($request, $id = null)
 
             $view = FOSRestView::create($context);
 
-            if (class_exists('FOS\RestBundle\Context\Context')) {
+            if (class_exists(Context::class)) {
```

Comment:
Import is missing

Reply:
It was already there https://github.com/sonata-project/SonataClassificationBundle/blob/3.x/src/Controller/Api/ContextController.php#L14